packages/utils/src/lighthouserc.js

```diff
@@ -42,17 +48,34 @@ function loadAndParseRcFile(pathToRcFile) {
 }
 
 /**
+ * Load file, parse and convert all `.` in key names to `:`
  * @param {string} pathToRcFile
  * @return {LHCI.LighthouseRc}
  */
 function loadRcFile(pathToRcFile) {
-  // Load the JSON and convert all `.` in key names to `:`
+  // Load file
   const contents = fs.readFileSync(pathToRcFile, 'utf8');
-  const rc = JSON.parse(contents);
+  const rc = parseFileContentToJSON(pathToRcFile, contents);
+  // Convert all `.` in key names to `:`
   recursivelyReplaceDotInKeyName(rc);
   return rc;
 }
 
+/**
+ * Parse file content to JSON.
+ * @param {string} pathToRcFile
+ * @param {string} contents
+ * @return {LHCI.LighthouseRc}
+ */
+function parseFileContentToJSON(pathToRcFile, contents) {
```

Comment:
If I may be nit-picky, there isn't any parsing to json, there is parsing from json or yaml. so I suggest to simply rename the method to `parseContent` and changing all the comments to reflect this

Reply:
No, you are not being nit-picky, it is great that you care about the name of things. Let me justify why I did choose the name. When content is passed to this function, it is one of two things: 1) a string yaml representation or 2) a string json representation. And the calls to yaml.safeLoad(contents) or JSON.parse(contents), is what parses the string to a javascript object. A more appropriate name would be `parseFileContentToJavaScriptObject` or just `parseFileContentToObject`, but since most devs know what JSON is and how it looks by calling it `parseFileContentToJSON`, I know devs will immediately imagine that the return of this function is something like: `{...}`. The problem that I see with `parseContent` is that by looking at the name I don't know what would be the result of this function. Will it be parsing content to `string`, `number`, `boolean`? Not sure by the name.

Reply:
I see, and mostly agree, but by the same logic I'd say ToJSON is misleading, because I wont get json as result from the method, you get an object. So by that logic I propose `parseContentToObject` or perharps `parseConfigFromContent`(would be really stretching it)

Reply:
Yeah, a `ToObject` would be the most appropriate. Or maybe `ToPOJO` jejeje. I don't like a lot the `parseConfigFromContent` because the function is doing nothing in regards of checking that the content is a valid config. Content could be `{ 'foo": "bla" }` and it would still parse that without problem.